### PR TITLE
Added eslint support and code cleaning

### PR DESCRIPTION
**Pull request by anikesh-butler**
Changed:
* Added client in client/client.js for local dev purposes.
* Added eslint and integrated with webpack
* Code cleaning

Testing:
* Tested locally in separate react project using npm link 

TODO: Add prettier and format the whole project after this MR

### CODE REVIEW

File: client/client.js [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+  if (docInfoState) {
+    submitLabels(docInfoState.trainingDocumentLabels);
+  }
+});
 
 // Step 1: Initialize Butler Blocks with your API Key
 
 // Get this API key from your Butler Account!
-const myApiKey = 'MY_API_KEY';
+const myApiKey = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJnb29nbGUtb2F1dGgyfDExNzI0MjgwNTUyNzczMTE1MTkzMyIsImVtYWlsIjoiYW5pa2VzaEBidXRsZXJsYWJzLmFpIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImlhdCI6MTY0NjgzODUxMjc2OX0.vOiBwdg41b3653AgGQc0-HKImZAwNdLYuaKh428ZZaU';
```
Comment (butler-matt) [inline client/client.js:15]: This should be reverted

File: client/client.js [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
 const butlerBlocks = loadButlerBlocks(myApiKey);
 
 // Step 2: Get your Document Info
 
 // Get this info from the API response when you upload your documents!
 const myDocument = {
-  modelId: 'MY_MODEL_ID',
-  documentId: 'MY_DOCUMENT_ID',
+  modelId: 'fd742066-f143-451d-ba73-b75c3340b670',
+  documentId: '3a06ddf2-23b2-4027-ac39-a8756577fe1b',
```
Comment (butler-matt) [inline client/client.js:24]: Ditto

File: client/index.html [hunk 3]
```diff
@@ -8,8 +8,11 @@
     <!-- We will put our Butler Block component inside this div. -->
     <div
       id="ButlerDocumentLabeler"
-      style="height: calc(100vh - 32px); width: calc(100vw - 32px); padding: 8px"
+      style="height: calc(100vh - 82px); width: calc(100vw - 32px); padding: 8px"
```
Comment (butler-matt) [inline client/index.html:11]: General thought on this /client folder. I've seen other repos do a structure like

/examples
--/example-1-name
--/example-2-name

This helps users easily see how to use the library for a variety of different use cases. Wdyt about doing something similar?

IE we could have

/examples
--/basic-usage
--/custom-save-button
--/react [Future]

etc.

File: client/client.js (hunk 1 above)
Comment (anikesh-butler) [reply client/client.js]: Done

File: client/index.html (hunk 3 above)
Comment (anikesh-butler) [reply client/index.html:11]: yeah, that is separate but the client directory is for us as a local dev client. The client also should be up to date with the latest changes for others to continue and make life a bit easier.

Comment (anikesh-butler) [reply client/index.html:11]: Technically it should be client as what we see from `npm start` is served from this directory.

Comment (anikesh-butler) [reply client/index.html:11]: The above examples directory you shared is for customers and should be ready to deploy not for the local dev env. I'll take that as a part of https://butlerinc.atlassian.net/browse/BS-4369.

Comment (butler-matt) [reply client/index.html:11]: Ahh I see, ok so this is only for local dev. Should it be in /src then? Will let you make the call on keeping it in /client vs /src

**Review by butler-matt — APPROVED** [review]

File: client/index.html (hunk 3 above)
Comment (anikesh-butler) [reply client/index.html:11]: the src directory is exclusive to the final bundled output. It's supposed to be out of src as it's not necessary for any change under the src directory.

**Pull request merged**